Source/WebCore/accessibility/AccessibilityNodeObject.cpp

```diff
@@ -2531,7 +2531,7 @@ static String accessibleNameForNode(Node& node, Node* labelledbyNode)
             StringBuilder passwordValue;
             passwordValue.reserveCapacity(inputValue.length());
             for (size_t i = 0; i < inputValue.length(); i++)
-                passwordValue.append(String::fromUTF8("•"));
+                passwordValue.append(String::fromUTF8("•"_s));
```

Comment:
Not good to put UTF-8 characters into an `ASCIILiteral`, since the character is not ASCII. This badness is newly introduced here. Some day we’ll add ASCII checking at compile time to `ASCIILiteral`, and then this will fail to compile.

Reply:
I assume this would hit assertions in debug. You can only use `_s` with ASCII:
```
    constexpr explicit ASCIILiteral(std::span<const char> spanWithNullTerminator)
        : m_charactersWithNullTerminator(spanWithNullTerminator)
    {
#if ASSERT_ENABLED
        for (size_t i = 0, size = length(); i < size; ++i)
            ASSERT_UNDER_CONSTEXPR_CONTEXT(isASCII(m_charactersWithNullTerminator[i]));
#endif
    }
```